### DISCUSSION TITLE
I keep getting an error when trying to remove gh-deploy from my github actions file

### DISCUSSION THREAD

**Discussion by stantios** [squidfunk/mkdocs-material#5264] [Q&A] [answered]
Description: I'm trying to add a new version to my docs which is 5.8.5 and this [issue](https://github.com/jimporter/mike/issues/140) said that I should remove gh-deploy once I've used it already first before I add a new version, so I tried that and then it gave me an error which I don't really understand.

Github Actions Error: https://github.com/Imzxhir/journeymap-docs/actions/runs/4525197127

**Comment by kamilkrzyskow** [comment]
The links returns a 404 error.
https://github.com/Imzxhir/journeymap-docs/actions
This page shows that you managed to build the site successfully.
Without the error description it's hard to tell what the issue was.

So, what was the issue or what was the solution?

**Comment by stantios** [comment, marked as answer]
Yeah, this issue was resolved. I had to add '- run: git pull' at the start of my run commands in the github actions file for it to work.
